velox/substrait/VeloxToSubstraitExpr.h

```diff
@@ -96,6 +97,12 @@ class VeloxToSubstraitExprConvertor {
 
   VeloxToSubstraitTypeConvertorPtr typeConvertor_;
 
+  /// Convert values in Velox complex vector to Substrait Literal.
+  void complexVectorToLiteral(
```

Comment:
Do not mix methods with member variables. Move this method above typeConvertor_. 

This method seems to be similar to toSubstraitLiteral methods, but has a different name and signature. Is this intentional?

Reply:
Yes, this is intentional.